application-admintools-default/src/main/java/com/xwiki/admintools/internal/data/ConfigurationDataProvider.java

```diff
@@ -57,9 +53,16 @@ public class ConfigurationDataProvider extends AbstractDataProvider
 
     private static final String TEMPLATE_NAME = "configurationTemplate.vm";
 
+    private final String dbName = "dbName";
```

Comment:
`private static final String DB_NAME = "name";`
same for version
Also, maybe you can use only `name` and `version`?

Reply:
I'd keep DB_NAME and DB_VERSION to keep it more specific.